dev/Dockerfile

```diff
@@ -13,8 +13,33 @@ RUN apk add --no-cache $BUILD_APKS inotify-tools \
 
 RUN chown -R zotonic /opt/zotonic/ebin
 
+# Install hdt tools
+
+WORKDIR /opt
+
+RUN apk --virtual build-deps add --update g++ make python curl automake autoconf libtool zlib-dev
+
+RUN curl -O http://download.drobilla.net/serd-0.28.0.tar.bz2 \
+    && tar jxf serd-0.28.0.tar.bz2 \
+    && rm serd-0.28.0.tar.bz2 \
+    && cd serd-0.28.0 \
+    && ./waf configure \
+    && ./waf \
+    && ./waf install
+
+RUN curl -L https://github.com/rdfhdt/hdt-cpp/archive/v1.3.2.tar.gz -o hdt-cpp-1.3.2.tar.gz \
+    && tar xzf hdt-cpp-1.3.2.tar.gz \
+    && rm hdt-cpp-1.3.2.tar.gz \
+    && cd hdt-cpp-1.3.2 \
+    && ./autogen.sh \
+    && PKG_CONFIG_PATH=/usr/local/lib/pkgconfig ./configure \
+    && make -j2 \
+    && make install
+
+RUN apk del build-deps
+
```

Comment:
These separate `RUN`s should all be combined into one: because of the way Docker layers work, removing the `build-deps` in a separate step won’t make the image smaller.